src/ReactHover.js

```diff
@@ -56,7 +56,7 @@ class ReactHover extends Component {
     }
 
     return (
-      <div>
+      <div style={{position: 'relative'}}>
```

Comment:
Is this change necessary for changing the behaviour when scrolling? if not, I don't think we can just make this decision for the user and not giving any override capability. @cht8687 let me know your thoughts here.